docs/sources/developers/http_api/playlist.md

```diff
@@ -133,6 +133,8 @@ Content-Type: application/json
 
 `GET /api/playlists/:uid/dashboards`
 
+/!\ Deprecated: this endpoint will be removed /!\
```

Comment:
Is user essentials ok with having their endpoint deprecated?

I think we should add support `dashboard_by_uid` in the backend until this can be sunset (not necessarily in this PR, but before we start populating playlists with `dashboard_by_uid` items). I believe we're still supporting deprecated endpoints until they're removed, which means we'll have to make sure it works as expected.

Reply:
That seems fine -- my goal here is really flagging these APIs as not for public consumption.  Given how this is used, i think we could remove this at anytime without notice, but seems like we should communicate that.

Reply:
With this PR, we no longer call the dashboards endpoint or the search function.  So seems like deprecating them now/soon is good.